Sprinkle of Yeezus/Settings.swift

```diff
@@ -12,67 +12,185 @@ import UserNotifications
+    
+    override func viewDidAppear(_ animated: Bool) {
+        //Loads saved time from datepicker when getting to view
+        let labelTime = UserDefaults.standard.string(forKey: "savedLabel")
+        informationLabel.text = labelTime
     }
     
     // MARK: - IBAction
-    @IBAction private func NotificationSwitch(_ sender: UISwitch) {
+    @IBAction private func notificationSwitch(_ sender: UISwitch) {
+        UserDefaults.standard.set(sender.isOn, forKey: "switchStatus") //saves the status of the switch
         if sender.isOn {
-            notificationCenter.requestAuthorization(options: [.alert, .sound], completionHandler: { didAllow, error in
+            notificationCenter.requestAuthorization(options: [.alert, .sound], completionHandler: { didAllow, error in /*
                 DispatchQueue.main.async {
                     sender.isOn = didAllow
                     self.updateUI()
-                }
+                } */
             })
         }
-        
         updateUI()
     }
     
-    @IBAction private func TimePicker(_ sender: UIDatePicker) {
-        sprinkleTimePicked = sender.date
+    @IBAction private func pickTimeAction() {
+        //Run when time is set on date picker
+        let storyboard = UIStoryboard(name: "Main", bundle: Bundle.main)
+        let controller = storyboard.instantiateViewController(withIdentifier: "TimePickerViewController") as! TimePicker
+        controller.delegate = self
+        navigationController?.pushViewController(controller, animated: true)
     }  
     
     // MARK: - Private
     private func updateUI() {
-        [DescLabel, DescButton].forEach { $0.isHidden = !Switch.isOn }
+        if notificationSwitch.isOn{
+            informationLabel.isHidden = false
+            changeTimeButton.isHidden = false
+        } else {
+            informationLabel.isHidden = true
+            changeTimeButton.isHidden = true
+        }
+        print("\(notificationSwitch.isOn)")
     }
     
+    
     private func sprinkleNotifications(_ sprinkleList: Array<Sprinkle>) {
         notificationCenter.removeAllPendingNotificationRequests()
         
-        guard Switch.isOn else {
+        guard notificationSwitch.isOn else { //Runs only user turns on notification switch
             return
         }
         
         let notification = UNMutableNotificationContent()
-        let notificationText = pickRandomQuote(sprinkleList)
-        notification.body = "\"\(notificationText.quote)\" \(notificationText.quoteSource), \(notificationText.date)"
-
-        let components = Calendar.current.dateComponents([.hour, .minute], from: sprinkleTimePicked)
-        let hour = components.hour ?? 12
-        let minute = components.minute ?? 0
+        var notificationText = [Sprinkle]()
+        var components = Calendar.current.dateComponents([.hour, .minute], from: sprinkleTimePicked) //grabs time from picker
+        
+        let today = Date()
+        let calendar = Calendar.current
```

Comment:
Move declaration of calendar to the top so that it can be used in the `components` variable above